packages/palette/src/elements/TextArea/TextArea.tsx

```diff
@@ -115,27 +101,33 @@ export const TextArea: React.ForwardRefExoticComponent<
             onChange={handleChange}
             defaultValue={defaultValue}
             required={required}
+            name={textAreaName}
+            title={title ? (typeof title === "string" ? title : "") : undefined}
```

Comment:
is there a reason why we only allow titles of the type string? 

Maybe instead of restricting titles to the type "string" here, what do you think about updating the type of the `TextArea`'s `title` prop instead?

Reply:
The reason behind this is that the `title` prop in the HTML textarea only accepts strings, but the `title` prop in the Palette Textarea component accepts ReactNode

I didn't want to update the type for the `title` because that might cause breaking changes for the TextArea to not accept React nodes anymore, but I believe updating the type would be fine, thank you for raising this! :)

Reply:
Yeah, it should be fine and this whole thing should be considered a breaking change so if we want to fix any interfaces to make them make more sense, now is the time to do it.